erigon-lib/commitment/hex_patricia_hashed.go

```diff
@@ -1724,6 +1735,8 @@ func (hph *HexPatriciaHashed) fold() (err error) {
 					counters.accLoaded++
 				}
 				if !cell.loaded.storage() && cell.storageAddrLen > 0 {
+					hph.metrics.LoadStorage.Add(1)
+					hph.metrics.Accounts.LoadStorageInc(hph.metrics.Accounts.currentPlainKey)
```

Comment:
you have already method Account(). 
What i do not like here - 2 lines which could be combined into one call of `.metrics.StorageLoaded(pk)` - so metrics will have consistent API. Not like one atomic is publicly available so i trigger it here and then put pk into function totally related to previous call but somewhy separated

Reply:
oh i found that there is already a `.Storage` method, why not used over here? or why `.Account()` is used above for tracing account loading?